### PR TITLE
Np 7685/transaction history bug temp fix

### PR DESCRIPTION
**Pull request by smirnovaka**

### CODE REVIEW

File: src/foam/comics/v2/DAOControllerConfig.js [hunk 1]
```diff
@@ -24,6 +26,15 @@ foam.CLASS({
   ],
 
   properties: [
+    {
+      class: 'StringArray',
+      name: 'order',
+      factory: function() { return []; }
```
Comment (kgrgreer) [inline src/foam/comics/v2/DAOControllerConfig.js]: This line is redundant. That's already what StringArray does.

**Comment by jlhughes** [comment]
How does this fix the issue?

**Comment by smirnovaka** [comment]
> How does this fix the issue?

it doesnt.  This is a part of temporary fix, along with [nanopay side](https://github.com/nanoPayinc/NANOPAY/pull/15287). We are still investigating the real cause

File: src/foam/comics/v2/DAOControllerConfig.js [hunk 2]
```diff
@@ -24,6 +26,14 @@ foam.CLASS({
   ],
 
   properties: [
+    {
+      class: 'StringArray',
+      name: 'order'
+    },
+    {
+      class: 'Boolean',
+      name: 'orderDesc'
```
Comment (kgrgreer) [inline src/foam/comics/v2/DAOControllerConfig.js]: Better to get rid of this and instead just have an optional '-' sign in front of properties you want to be descending.
Ex.: firstName,lastName,-age

**Pull request merged**